src/replication.c

```diff
@@ -2679,21 +2689,26 @@ replicate(ISTGT_LU_DISK *spec, ISTGT_LU_CMD_Ptr cmd, uint64_t offset, uint64_t n
 				goto retry_read;
 			}
 
+
+			timesdiff(CLOCK_MONOTONIC, queued_time, now, diff);
 			count = 0;
-			if (success_resp == true) {
```

Comment:
success_resp is not required now

Reply:
yes

Reply:
success_resp variable need to be removed. Its not removed yet.

Reply:
ok